scripts/build.ps1

```diff
@@ -1102,6 +1109,20 @@ function Build-SpecificProjects
     }
 }
 
+function Test-GitChanges { 
+    param (
+        [string[]]$Like
+    )
+
+    # For each line check if we match any of the rules
+    # if we do select that path. We do Where-Object twice beacuse 
+    # -like does not support multiple patterns on the right hand side. 
+    # We could check if any pattern matches any line, but that returns
+    # the patterns that match any path, and not paths that return any pattern. 
+    # So the current approach is better for debugging.
+    $gitChanges | Where-Object { $path = $_; $like | Where-Object { $path -like $_ }}
+}
```

Comment:
```suggestion
    # For each line check if we match any of the rules.
    # We do Where-Object twice because  -like does not support multiple 
    # patterns on the right hand side. 
    # We could check if any pattern matches any line, with simpler code, 
    # but that returns the patterns that matched any path, and not paths 
    # that matched any pattern.
    # So the current approach is better for debugging.
    $gitChanges | Where-Object { $path = $_; $like | Where-Object { $path -like $_ }}
}
```